maven-extension/src/main/java/io/opentelemetry/maven/OpenTelemetrySdkService.java

```diff
@@ -25,6 +25,9 @@
 @Component(role = OpenTelemetrySdkService.class, hint = "opentelemetry-service")
 public final class OpenTelemetrySdkService implements Initializable, Disposable {
 
+  public static final String VERSION =
+      OpenTelemetrySdkService.class.getPackage().getImplementationVersion();
```

Comment:
I don't think we want to expose this as a public field

Reply:
Good point @anuraaga, I'd missed that.

Reply:
Thanks, PR created https://github.com/open-telemetry/opentelemetry-java-contrib/pull/198